devs/docs/release.rst

```diff
@@ -105,5 +105,21 @@ archived.
 You can archive releases by selecting *Edit*, unselecting the *Active*
 checkbox, and then saving.
 
+Release Notes 
+=======================
+
+Once a release is done, we create a release note, which is
```

Comment:
The release notes are just moved from unreleased to the tagged release. I think this is already documented somewhere. 

I don't think that re-writing the changes should be part of the release process. This should be done prior to a release whenever we make a user facing change.

Reply:
Thanks for your feedback, @mfussenegger, will delete this PR as discussed.